report/Process/OutSideChanges.tex

```diff
@@ -0,0 +1,21 @@
+\section{Outside changes}
+During the March the group, along with the rest of the country was effected by the Corona Virus.
+Along with the rest of the country the university was shutdown with short notice.
+During this period none of the group members was able to collect any personal belongings within the group room and the effect of their "home base" was thus no longer possible in a shared location.
+One could argue, that by moving the workplace from the university to the home, would make the "home base" more profound, but however multiple group members felt that always being within their home, made the lines for when to work and when to relax more blurry.
+Whilst working from home, each group member experienced different challenges, such as missing motivation, kids not able to attend daycare, etc.
+As an effect hereof the productivity of each group member fell drastically.
+As another effect of the initial stages of the Corona Virus, the uncertainty of the courses and exams layed another concern in the minds of each group member as the university created new guidelines for both.
+
+With this new "normal", the group could slowly regain their ability to work and heighten their productivity.
+In order to work using the Essence working process, each of the group members was assigned a role.
+While working remotely each of these roles was used in varying degrees.
+Some roles came naturally, while others needed a little more work to be used.
+The ability to use the role as a child and challenger would prove difficult to perform from a distance and while not being physically located close to one another. 
+
+With the use online meeting tools, such as Discord and Microsoft Teams, the group would facilitate and participate in meetings on a regular basis.
+While the tools was very useful for maintaining communications, the use of a GitHub was crucial to communicate requirements and features, which worked perfectly for each group member, since they where used to work with GitHub for version control.
```

Comment:
```suggestion
While the tools was very useful for maintaining communications, the use of GitHub was crucial to communicate requirements and features, which worked perfectly for each group member, since they used to work with GitHub for version control.
```